recipe-server/normandy/recipes/api/serializers.py

```diff
@@ -67,6 +88,8 @@ class RecipeSerializer(serializers.ModelSerializer):
     filter_expression = serializers.CharField(read_only=True)
     latest_revision_id = serializers.CharField(source='latest_revision.id', read_only=True)
     approved_revision_id = serializers.CharField(source='approved_revision.id', read_only=True)
+    latest_revision = RecipeRevisionSerializer(read_only=True)
+    approved_revision = RecipeRevisionSerializer(read_only=True)
```

Comment:
I might by missing something, but it seems like we declare these to be denormalized, and then everywhere we use them, we replace the denormalized data with an ID. Could we make these `latest_revision_id` and `approved_revision_id` instead?